### PR TITLE
Jakarta packages for the persistent executor

### PR DESCRIPTION
**Pull request by njr-11**
Apply the Jakarta transformer for the persistent executor and at least one test bucket to demonstrate that it is working for core scenarios.

### CODE REVIEW

**Comment by njr-11** [comment]
#build
This might require a fix that is under #13721

**Comment by skasund** [comment]
L2 message review not required

File: dev/com.ibm.websphere.appserver.features/visibility/public/persistentExecutor-2.0/resources/l10n/com.ibm.websphere.appserver.persistentExecutor-2.0.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the creation of persistent scheduled executors that allow applications \
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/persistentExecutor-2.0/resources/l10n/com.ibm.websphere.appserver.persistentExecutor-2.0.properties:17]: Suggested edit:

This feature enables the creation of persistent scheduled executors that allow applications \
 to schedule persistent tasks that run with _a_ thread context that is managed by the application server.

Comment (dmuelle) [reply dev/com.ibm.websphere.appserver.features/visibility/public/persistentExecutor-2.0/resources/l10n/com.ibm.websphere.appserver.persistentExecutor-2.0.properties:17]: add article "a" before "thread context"

Comment (njr-11) [reply dev/com.ibm.websphere.appserver.features/visibility/public/persistentExecutor-2.0/resources/l10n/com.ibm.websphere.appserver.persistentExecutor-2.0.properties:17]: This description was copied from the corresponding v1.0 feature, so I'm fixing it as you suggest in both places. Please see https://github.com/OpenLiberty/open-liberty/pull/13732/commits/d58aee9651a916c90c362bf4e9f98d4a2d03017f for the change

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
Build njr-11-13732-20200923-2157 can be used for delivery.  I had submitted another build after that because the packaging verification build step hadn't started (it happens asynchronously).  However, it eventually did start and completed successfully.  The build otherwise has 3 test failures, which are duplicates of pre-existing issues.

**Comment by tkburroughs** [comment]
@njr-11 Merged

**Pull request merged**